README.md

```diff
@@ -141,12 +141,37 @@ build_windows_cuda.bat
 Build can take ~2 hours depending on the machine configuration.
 
 ### Build and test using docker
+
+### Prerequisites
+* Ensure virtualization is enabled in both the BIOS settings of your computer and the Windows virtualization feature
+* Install WSL 2 on your system
```

Comment:
Is WSL2 a requirment for running docker ?

Reply:
not necessarily but it is good if it is there